test/rapidgen/rapidgenerators.go

```diff
@@ -0,0 +1,55 @@
+package rapidgen
```

Comment:
Please move this into the network test package. Files that do not end with `_test.go` are visible to non test code and could end up in the production binary. There isn't a good way to make test specific cross-package utilities in go. There have been a number of golang improvement proposals related to this pattern, [here's one I made which has some suggestions about the current idioms](https://github.com/golang/go/issues/44086).

Reply:
Thanks Will